concepts/objects/introduction.md

```diff
@@ -0,0 +1,129 @@
+# Introduction
```

Comment:
Imo the headers from the about should be in this doc, too. Makes it easier to navigate since it's quite long.

Reply:
You are totally right. Fixed.